bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/DataClientVeneerApi.java

```diff
@@ -134,6 +140,18 @@ public Result apply(Row row) {
         MoreExecutors.directExecutor());
   }
 
+  @Override
+  public ResultScanner readRows(Query.QueryPaginator paginator, int requestedPageSize) {
```

Comment:
Maybe a better way to do this is add a getPageSize() method to QueryPagninator so we don't need to pass in the requestedPageSize here again. Can you make a change to https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Query.java#L349 and mark the getPageSize() method to be `@InternalApi`?

Reply:
https://github.com/googleapis/java-bigtable/pull/1987